include/scrimmage/entity/External.h

```diff
@@ -178,7 +178,8 @@ class External {
         boost::optional<std::list<NetworkDevicePtr>> subs =
             pubsub_->find_subs(network_name, topic_name);
 
-        auto no_subs_msg = [&]() {
+        auto no_subs_msg = [=]() { //capture by copy, used in returned function.
+            //Note: Find_subs already prints this information. Keep it here or there?
```

Comment:
If it is already printed in find_subs, can we just make this function empty?

Reply:
Idk. I would, but it wasn't like that before, so you decide. Maybe you'd prefer to remove it from the other location, how'd you want to design it. You wrote this didn't you?